packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -489,6 +498,7 @@ const UnifiedDesignPicker: React.FC< UnifiedDesignPickerProps > = ( {
 					purchasedThemes={ purchasedThemes }
 					currentPlanFeatures={ currentPlanFeatures }
 				/>
+				{ ! isShowAll && bottomAnchorContent }
```

Comment:
If the user never selected any vertical, the event won't be triggered under the "Show All" category

Reply:
> BTW, could you help to confirm what's the meaning of the end here? thanks!

I tried to avoid duplication when user selects 'Show all' https://github.com/Automattic/wp-calypso/blob/9abb24c43b0c4a1f470f6a2775e32ccf0c283a6c/packages/design-picker/src/components/unified-design-picker.tsx#L521
But as you pointed out, the anchor has disappeared when user not select any vertical, I'll look at this